src/test/java/com/loofah/graph/api/helpers/IntegrationTestHelpers.java

```diff
@@ -44,4 +48,10 @@ public static void assertSkillHasGradeWithTitle(LinkedHashMap skill, String expe
         LinkedHashMap gradeOnSkill = (LinkedHashMap) skill.get(SkillDTO.SkillDTOFields.GRADE.key());
         assertEquals(expectedGradeTitle, gradeOnSkill.get("title"));
     }
+
+    public static void assertSkillHasCraftWithTitle(LinkedHashMap skill, List<String> expectedCraftTitles){
```

Comment:
[tiny] method name should be plural to show a skill has a list of crafts: `assertSkillHasCraftsWithTitles`

Reply:
updated :)